cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -152,7 +152,7 @@ auto Model::_createBindGroupLayout(wgpu::Device device, const std::vector<Unifor
     -> wgpu::BindGroupLayout {
   std::vector<wgpu::BindGroupLayoutBinding> bindings;
   for (uint32_t i = 0; i < uniforms.size(); i++) {
-    auto binding = wgpu::BindGroupLayoutBinding{i, wgpu::ShaderStage::Vertex, wgpu::BindingType::UniformBuffer,
+    auto binding = wgpu::BindGroupLayoutBinding{i, uniforms[i].shaderStage, wgpu::BindingType::UniformBuffer,
```

Comment:
I wonder if we could just default this to always assign to both, so apps don't need to track this? Or do we get warnings and inefficiencies if we assign it but then it is not used?

Reply:
I'm not what the implications of doing that would be, if there's an additional overhead (I assume there is, else it would be done by default?) I'd prefer we declare it explicitly. If not we can of course get rid of it